app.R

```diff
@@ -457,9 +456,7 @@ schema_to_display_lookup <- data.frame(schema_name, display_name)
     template_type_df <- schema_to_display_lookup[match(input$template_type, schema_to_display_lookup$display_name), 1, drop = F ]
     template_type <- as.character(template_type_df$schema_name)
 
-    annotation_status <- metadata_model$validateModelManifest(input$file1$datapath, template_type) %>% 
-                          rapply(., function(x) gsub("\'\'", "\' \'", x), how = "replace") %>% # change '' to ' '
-                          rapply(., function(x) gsub("^$", " ", x), how = "replace") # change empty value to (one space) blank
+    annotation_status <- metadata_model$validateModelManifest(input$file1$datapath, template_type) 
```

Comment:
Only change `"NA"` to `""` in #L420 can solve the issue, no need to change both to one space